docs/integrations/github_actions/aws_oidc.md

```diff
@@ -0,0 +1,172 @@
+
+In order to execute the Terraform code and deploy the resources GitHub needs, you will need local credentials for the target AWS account. This can be via AWS Identity Center, IAM User Access Keys, or via environment variables. Regardless of how you [authenticate](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration), the permissions required to deploy the Terraform code must have permission to create IAM resources.
+
+### Configuration
+
+Update the `default.tfvars` file for the below variables.
+
+* `github_repo`: GitHub repository that needs the access token. Example: octo-org/octo-repo
+* `github_branch`: GitHub branch that runs the workflow. If you plan to trigger the workflow through schedule, then this must be the default branch. If you plan to run the workflow manually, this can be any branch. Example: master
+* `aws_iam_role_name`: Name of the AWS IAM Role to create. Example: steampipe_gh_oidc_demo
+
+### Implementation
+
+Run these commands to create resources in your AWS Account.
+
+* `terraform init`: Initialize Terraform to get all necessary providers.
+* `terraform apply -var-file=default.tfvars`: Apply the configuration using the configuration file "defaults.tfvars"
+
+### Verification
+
+Terraform creates an Identity provider and IAM Role in your account.
+
+* AWS > IAM > Identity provider > token.actions.githubusercontent.com
+* AWS > IAM > Role (rolename: steampipe_gh_oidc_demo)
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"90%"}}>
+<img alt="aws_iam_identity_provider" src="/images/docs/ci-cd-pipelines/oidc/aws_iam_identity_provider.png" />
+</div>
+
+<div style={{"marginBottom":"2em","borderWidth":"thin", "borderStyle":"solid", "borderColor":"lightgray", "padding":"20px", "width":"90%"}}>
+<img alt="aws_iam_role" src="/images/docs/ci-cd-pipelines/oidc/aws_iam_role.png" />
+</div>
+
+You will need to add one GitHub Secret, `OIDC_AWS_ROLE_TO_ASSUME`, which is the ARN of the above IAM role that handles OIDC federation. You can get the IAM Role ARN from the Terraform output or the AWS IAM Console.
```

Comment:
First, this seems to be a required step, but its kind of hidden in a section called *Verification* - This seems to be a required step, not just verification that things were created.  Second, it probably needs to be a little more specific what needs to be done.  The previous screenshoots look like the aws console, and suddenly we are in the github UI ...